source/stylesheets/refills/_laptop.scss

```diff
@@ -0,0 +1,73 @@
+.laptop {
+  $lid-background: darken(#fff, 5);
+  $inset-shadow: inset 0 0px 1px transparentize(#000, 0.8);
```

Comment:
Color literals like `#000` should only be used in variable declarations; they should be referred to via variable everywhere else.<br>`0px` should be written without units as `0`